change/@fluentui-azure-themes-7de5e42b-cf77-4e35-a9a0-9c9b05b49a37.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@spmonahan Is it correct that there are about 59 files added under `change` folder by `yarn change`?

Reply:
That seems like a bug. I'd expect to have 2: one for `react` and one for `utilities`.

Try reverting the commit with the change files and pulling the latest changes from `upstream master` into your branch.

Reply:
yeah, that was what I expected. update my fork seems fix it.